lib/matplotlib/backends/backend_svg.py

```diff
@@ -1235,7 +1235,11 @@ def _print_svg(self, filename, svgwriter, fh_to_close=None, detach=False,
             renderer.finalize()
         finally:
             if detach:
-                svgwriter.detach()
+                if six.PY3:
+                    svgwriter.detach()
+                else:
+                    svgwriter.reset()
+                    svgwriter.stream = io.BytesIO()
```

Comment:
This is not intended to fix #6870. It is the Python 2 equivalent of 946367f, which fixes #6926.

#6870 is fixed by 7fe62ac, because the entire cached renderer (not just this part) is not supposed to be pickled.